libs/openFrameworks/types/ofBaseTypes.h

```diff
@@ -448,20 +448,27 @@ class ofBaseVideoGrabber: virtual public ofBaseVideo{
 class ofBaseVideoPlayer: virtual public ofBaseVideo{
 
 public:
+	/// \brief Destroys the ofBaseVideoPlayer.
 	virtual ~ofBaseVideoPlayer();
 
-	/// \brief Load a video at the file path specified by \p name.
-	/// \param name The file path of the video you would like to load.
-	/// \returns Returns true if the video was loaded successfully.
-	/// \sa loadAsync
+	/// \brief Load a video resource by name.
```

Comment:
If the implementation implements it that is :)  I had to dig deep into that yesterday and realized that ...